src/app/modules/item/pages/item-edit/item-edit.component.ts

```diff
@@ -86,11 +87,20 @@ export class ItemEditComponent implements OnDestroy, PendingChangesComponent {
         this.initialFormData = data;
         this.resetForm();
       });
+
+    this.maxTeamSizeSubscription = this.itemForm
+      .get('entry_participant_type')
+      ?.valueChanges
```

Comment:
usually we display these on a single line
```suggestion
      .get('entry_participant_type')?.valueChanges
```